src/components/home/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+import { menuHeightOffset, Title1, SubTitle1, Button, Arrow } from '../styledComponents';
+import headerImage from '../../assets/homePage/headerImage.jpg';
+
+const HeaderContainer = styled(Grid)({
+  height: `${680 - menuHeightOffset}px`,
+});
+
+const ImageContainer = styled('div')({
+  position: 'absolute',
+  width: 'calc(100vw - 16px)',
+  left: '0px',
+  top: '0px',
+  zIndex: '-1',
+});
+
+const Content = styled(Grid)({
+  width: '50%',
+});
+
+const ButtonContainer = styled(Grid)({
+  margin: '51px 0px 104px',
+});
+
+
+const ArrowContainer = styled(Grid)({
+  cursor: 'pointer',
+  marginBottom: '91px',
+});
+
+const HeaderImage = styled('img')({
+  width: '100%',
+  height: '680px',
+});
+
+const Header = () => (
```

Comment:
Especially since `header` is an official HTML component, I usually think of `Header` as being an element of a page layout that is present on every page (e.g. the navbar), rather than the topmost element of a page. Maybe rename this component to something like `Hero`?